### PR TITLE
Update az find service endpoint

### PR DESCRIPTION
**Pull request by xinshiMSFT**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by xinshiMSFT** [comment]
Can you please review and give us a heads up before merging? We want to make sure our service is ready by then.

**Comment by Juliehzl** [comment]
Can you try to fix the[ CI error](https://dev.azure.com/azure-sdk/public/_build/results?buildId=98945&view=logs&j=bbb91ccf-96ce-5c28-8d12-446dc22f639b&t=97f87583-ed5f-576c-dcfa-8e2962b1cccd&l=6908) first? I think you should update your test scenario code according to you code change in command.

**Comment by Juliehzl** [comment]
One more thing to note is can you try to deal with `''`  issue for windows OS. For example, in windows `az find 'az storage'` cannot work, but `az find "az storage"` can. And why I get HTTP connection error based on your code when I run `az find` :
```
(env) E:\Microsoft\AzureCLI\azure-cli>az find "az storage"
Finding examples...
The command failed with an unexpected error. Here is the traceback:

HTTPSConnectionPool(host='app.aladdin.microsoft.com', port=443): Max retries exceeded with url: /api/v1.0/examples?query=az+storage&clientType=AzureCli&context=%7B%22sessionId%22%3A+%22ec46785e-cad6-4e67-b388-e10f6d3e4176%22%2C+%22subscriptionId%22
%3A+%220b1f6471-1bf0-4dda-aec3-cb9272f09590%22%2C+%22clientRequestId%22%3A+%22052aef48-caec-11e9-a7b0-186024942c80%22%2C+%22installationId%22%3A+%220a80c41e-9646-11e9-8507-186024942c80%22%2C+%22versionNumber%22%3A+%222.0.71%22%7D (Caused by SSLErro
r(SSLCertVerificationError("hostname 'app.aladdin.microsoft.com' doesn't match either of '*.azurewebsites.net', '*.scm.azurewebsites.net', '*.sso.azurewebsites.net', '*.azure-mobile.net', '*.scm.azure-mobile.net'")))
Traceback (most recent call last):
...
```
Have you tried it?

File: src/azure-cli/azure/cli/command_modules/find/commands.py [hunk 1]
```diff
@@ -6,5 +6,5 @@
 
 def load_command_table(self, _):
     with self.command_group('') as g:
-        g.custom_command('find', 'process_query', is_preview=True)
+        g.custom_command('find', 'process_query', is_preview=False)
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/find/commands.py]: If you want to remove Preview flag, you should update history.rst to mark `az find` in GA.

**Pull request closed without merging**